activemodel/lib/active_model/errors.rb

```diff
@@ -205,17 +208,37 @@ def [](attribute)
       DeprecationHandlingMessageArray.new(messages_for(attribute), self, attribute)
     end
 
-    # Iterates through each error key, value pair in the error messages hash.
+    def first
+      deprecation_index_access_warning(:first)
```

Comment:
FYI this is broken, `deprecation_index_access_warning` expects 2 arguments

Reply:
Good point. I removed this deprecation since it is still a breaking change.

Reply:
@Edouard-chin my bad, thanks for catching this.